### PR TITLE
fozzie-components@v3.36.0 - Added additional circleci cli tasks 

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
v3.36.0
------------------------------
*May 27, 2021*

## Added
- CircleCI config for running `build`, `lint` & `unit testing` steps individually locally.
- Documentation into storybook guides as to how to run these tests through CircleCI CLI locally and what each task is setup to do.

### CODE REVIEW

**Review by LTurns — DISMISSED** [review]

**Review by LTurns — DISMISSED** [review]

File: .vscode/tasks.json [hunk 1]
```diff
@@ -2,7 +2,23 @@
     "version": "2.0.0",
     "tasks": [{
         "label": "CircleCI CLI",
```
Comment (siggerzz) [inline .vscode/tasks.json]: Rename this to `CircleCI Complete Build`

File: .vscode/tasks.json [hunk 2]
```diff
@@ -2,7 +2,23 @@
...
+        "command": "circleci local execute --config .circleci/local_config.yml --job build",
+        "type": "shell"
+    },
+    {
+        "label": "CircleCI Lint",
+        "command": "circleci local execute --config .circleci/local_config.yml --job lint-build",
+        "type":"shell"
+    },
+    {
+        "label": "CircleCI Unit Tests",
```
Comment (siggerzz) [inline .vscode/tasks.json]: Rename to `CircleCI Unit + Integration Tests`

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v3.35.0
```
Comment (siggerzz) [inline CHANGELOG.md]: This will need to be 3.36

File: stories/guides/circlieci-cli.stories.mdx [hunk 4]
```diff
@@ -44,7 +44,14 @@ Once you have installed the CircleCI CLI, you can simply open your VScode, and p
...
 and type; `‘Run task’`. 
 
-Hit enter and you will then see the `CircleCI CLI task` pop up. 
-Just hit enter again and then once more on the option, `"Continue without scanning the task output"` to run the CircleCI build. 
+Hit enter and you will then see a list of `CircleCI` tasks.
 
-This can take around 15-20 mins to complete the run.
\ No newline at end of file
+Currently we have (and what they run);
+`CircleCI CLI` - Full CircleCI workflow
```
Comment (siggerzz) [inline stories/guides/circlieci-cli.stories.mdx]: Make sure to update the name here too

**Review by siggerzz — CHANGES_REQUESTED** [review]
Few minor comments, but looking good!

**Review by LTurns — DISMISSED** [review]

File: stories/guides/circlieci-cli.stories.mdx [hunk 5]
```diff
@@ -44,7 +44,14 @@ Once you have installed the CircleCI CLI, you can simply open your VScode, and p
...
-Just hit enter again and then once more on the option, `"Continue without scanning the task output"` to run the CircleCI build. 
+Hit enter and you will then see a list of `CircleCI` tasks.
 
-This can take around 15-20 mins to complete the run.
\ No newline at end of file
+Currently we have (and what they run);
+`CircleCI Complete Build` - Full CircleCI workflow
+`CircleCI Build` - Build steps of the CircleCI workflow
+`CircleCI Lint` - Linting steps of the CircleCI workflow
+`CircleCI Unit` - Build and unit testing steps of the CircleCI workflow
```
Comment (siggerzz) [inline stories/guides/circlieci-cli.stories.mdx]: Update the build name here to be `CircleCI Unit + Integration` - The description also needs to be updated, as this doesn't do a build.

**Review by siggerzz — CHANGES_REQUESTED** [review]
Minor comments

File: stories/guides/circlieci-cli.stories.mdx (hunk 5 above)
Comment (JoshuaNg2332) [reply stories/guides/circlieci-cli.stories.mdx]: Updated this. But we need to build for some unit tests to pass.

**Review by siggerzz — DISMISSED** [review]

**Review by LTurns — DISMISSED** [review]

**Review by siggerzz — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**